docs/labwc-theme.5.scd

```diff
@@ -126,6 +126,28 @@ This syntax is not documented on the openbox.org wiki, but is supported by
 openbox and is used by many popular themes. For the sake of brevity, these
 elements are not listed here, but are supported.
 
+*window.active.shadow.inset*
+	How far behind the focused window should its drop-shadow be inset, as a
+	proportion of shadow radius. This determines the corner radius of the
+	shadow.
```

Comment:
All descriptions are missing the default values which also clears up the unit expected here, e.g. something like `Default is 0.5.`